presto-native-execution/presto_cpp/presto_protocol/tests/ClpConnectorProtocolTest.cpp

```
@@ -0,0 +1,189 @@
+
+  std::shared_ptr<ConnectorTableLayoutHandle> handle;
+  protocol.deserialize(binaryData, handle);
+
+  auto clpHandle = std::dynamic_pointer_cast<ClpTableLayoutHandle>(handle);
+  ASSERT_NE(clpHandle, nullptr);
+  EXPECT_EQ(clpHandle->table.schemaTableName.schema, "clp");
+  EXPECT_EQ(*clpHandle->kqlQuery, "level == 'error'");
+  EXPECT_EQ(*clpHandle->metadataFilterQuery, "timestamp > 1000");
+}
+
+TEST_F(ClpConnectorProtocolTest, TestSplitDeserializationArchive) {
```

Clang (14.0.6)

[warning] invalid case style for function 'TEST_F'

(readability-identifier-naming)